### DISCUSSION TITLE
How can I restore wiki from a daily backup?

### DISCUSSION THREAD

**Discussion by jord8on** [requarks/wiki#2902] [Help / Questions] [answered]
Within Admin > Storage > Local File System there is an option to "Create Daily Backups"

We have this toggled on but cannot find instructions on how to restore from one of these daily backups. 

Can someone please advise?

Screenshot:

![image](https://user-images.githubusercontent.com/20749549/104082066-44034d00-51f0-11eb-8e32-72952ca3ad25.png)

**Comment by NGPixel** [comment, marked as answer]
You would extract the backup file into a dedicated directory.

Then configure the Local File System storage module to point to that directory. Finally click the "Import Everything" action.
